Scripts/build-webdriveragent.js

```diff
@@ -25,7 +28,7 @@ async function buildWebDriverAgent (xcodeVersion) {
   // Clean and build
   try {
     await exec('/bin/bash', ['./Scripts/build.sh'], {
-      env: {TARGET: 'runner', SDK: 'sim', DERIVED_DATA_PATH},
+      env: {TARGET: process.env.TARGET, SDK: process.env.SDK, DERIVED_DATA_PATH},
```

Comment:
there should be some check first if these env variables are porvided

Reply:
9ea050b272838c41d672e7550096b67d889465f5